### PR TITLE
Feature: Adjust lighthouse congfigs

### PR DESCRIPTION
**Pull request by Wolmin**
With those configs we will be able to correctly run lighthouse validator on mainnet/testnet/devnet networks

### CODE REVIEW

File: devnets/7420/lighthouse/validator.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+testnet-dir = "./configs/devnet/shared"
+validators-dir = "./devnet-keystore/validators"
+secrets-dir = "./devnet-keystore/validators"
+beacon-nodes = "http://127.0.0.1:4000"
+init-slashing-protection = true
+http = true
+http-port = 5062
+metrics = true
+metrics-address="127.0.0.1"
+metrics-allow-origin="*"
```
Comment (mxmar) [inline devnets/7420/lighthouse/validator.toml:10]: Please remove it.

```
 .help("Set the value of the Access-Control-Allow-Origin response HTTP header. \
                    Use * to allow any origin (not recommended in production). \
                    If no value is supplied, the CORS allowed origin is set to the listen \
                    address of this server (e.g., http://localhost:5054).")
```

File: mainnet/lighthouse/lighthouse.toml [hunk 2]
```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
 listen-address = "0.0.0.0"
```
Comment (mxmar) [inline mainnet/lighthouse/lighthouse.toml:6]: How many `inbound` peers were present in test scenario?

https://github.com/sigp/lighthouse/blob/e9a7316f1de113d118f1aa16856ae97e181ceb82/beacon_node/src/cli.rs#L193 -> IDK if `0.0.0.0` is not an issue here

File: testnet/lighthouse/lighthouse.toml [hunk 3]
```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
...
 enable-private-discovery = true
 enr-address = "0.0.0.0"
 metrics = true
-metrics-address = "0.0.0.0"
+metrics-address = "127.0.0.1"
 metrics-allow-origin = "*"
 metrics-port = 5054
-boot-nodes = 'enr:-MK4QHfHBBWZqSIxumgMmdC3zd7QyJD2shEXBhSKbay3aYwdAHrhb3lksHStJPk1tLoFOPaU4ELtaahRGUUpfm5tmhCGAYfiNvG0h2F0dG5ldHOIAAAAAAAAAACEZXRoMpBWKoojQgEABCoAAAAAAAAAgmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQOLhdJYX4WgjYXPw6GpCkuxnNWvM0KE_aeYzek95q2f-IhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A'
\ No newline at end of file
+boot-nodes = 'enr:-MK4QPAebhGrV0IdXCaVQ33cL-5oySUvBXPkXZTHhCDrqMOtITlhab5B6PL9hfQY5NuRF5ctLcSFhHMCdLT4RWxxo-SGAYiVi7DIh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCvClM1QgEABP__________gmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQISdsDnr_lxpAXRQmeOu-F_eAtxypZI4ctFavqdagbka4hzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A'
```
Comment (mxmar) [inline testnet/lighthouse/lighthouse.toml:20]: Please update `prysm.yml` testnet ENR also. There are two, one needs to be changed. `enr:-MK4QHfHBBWZqSIxumgMmdC3zd7QyJD2shEXBhSKbay3aYwdAHrhb3lksHStJPk1tLoFOPaU4ELtaahRGUUpfm5tmhCGAYfiNvG0h2F0dG5ldHOIAAAAAAAAAACEZXRoMpBWKoojQgEABCoAAAAAAAAAgmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQOLhdJYX4WgjYXPw6GpCkuxnNWvM0KE_aeYzek95q2f-IhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A` is present in `prysm.yml`, change to: `enr:-MK4QPAebhGrV0IdXCaVQ33cL-5oySUvBXPkXZTHhCDrqMOtITlhab5B6PL9hfQY5NuRF5ctLcSFhHMCdLT4RWxxo-SGAYiVi7DIh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCvClM1QgEABP__________gmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQISdsDnr_lxpAXRQmeOu-F_eAtxypZI4ctFavqdagbka4hzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`

File: testnet/lighthouse/lighthouse.toml [hunk 4]
```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
...
 http-port = 4000
 enr-udp-port = 9000
 enr-tcp-port = 9000
 discovery-port = 9000
 enable-private-discovery = true
 enr-address = "0.0.0.0"
 metrics = true
-metrics-address = "0.0.0.0"
+metrics-address = "127.0.0.1"
 metrics-allow-origin = "*"
```
Comment (mxmar) [inline testnet/lighthouse/lighthouse.toml:18]: I think we can remove it (previous comment explained it).

File: testnet/lighthouse/validator.toml [hunk 5]
```diff
@@ -4,4 +4,8 @@ secrets-dir = "./testnet-keystore/validators"
 beacon-nodes = "http://127.0.0.1:4000"
 init-slashing-protection = true
 http = true
-http-port = 5062
\ No newline at end of file
+http-port = 5062
+metrics = true
+metrics-address="127.0.0.1"
+metrics-allow-origin="*"
```
Comment (mxmar) [inline testnet/lighthouse/validator.toml:10]: I think we can remove it (previous comment explained it).

File: mainnet/lighthouse/validator.toml [hunk 6]
```diff
@@ -4,4 +4,8 @@ secrets-dir = "./mainnet-keystore/validators"
 beacon-nodes = "http://127.0.0.1:4000"
 init-slashing-protection = true
 http = true
-http-port = 5062
\ No newline at end of file
+http-port = 5062
+metrics = true
+metrics-address="127.0.0.1"
+metrics-allow-origin="*"
```
Comment (mxmar) [inline mainnet/lighthouse/validator.toml:10]: I think we can remove it (previous comment explained it).

File: testnet/lighthouse/lighthouse.toml [hunk 7]
```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
...
 eth1 = true
 http = true
-http-address = "0.0.0.0"
+http-address = "127.0.0.1"
 http-port = 4000
 enr-udp-port = 9000
 enr-tcp-port = 9000
 discovery-port = 9000
 enable-private-discovery = true
 enr-address = "0.0.0.0"
```
Comment (mxmar) [inline testnet/lighthouse/lighthouse.toml:15]: Same here: https://github.com/sigp/lighthouse/blob/e9a7316f1de113d118f1aa16856ae97e181ceb82/beacon_node/src/cli.rs#L193

File: mainnet/lighthouse/lighthouse.toml [hunk 8]
```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
...
 eth1 = true
 http = true
-http-address = "0.0.0.0"
+http-address = "127.0.0.1"
 http-port = 4000
 enr-udp-port = 9000
 enr-tcp-port = 9000
 discovery-port = 9000
 enable-private-discovery = true
 enr-address = "0.0.0.0"
```
Comment (mxmar) [inline mainnet/lighthouse/lighthouse.toml:15]: Same here: https://github.com/sigp/lighthouse/blob/e9a7316f1de113d118f1aa16856ae97e181ceb82/beacon_node/src/cli.rs#L193

**Review by mxmar — CHANGES_REQUESTED** [review]
I've left comments @Wolmin

File: devnets/7420/lighthouse/validator.toml (hunk 1 above)
Comment (Wolmin) [reply devnets/7420/lighthouse/validator.toml]: Done

File: testnet/lighthouse/lighthouse.toml (hunk 3 above)
Comment (Wolmin) [reply testnet/lighthouse/lighthouse.toml:19]: Done

File: testnet/lighthouse/validator.toml (hunk 5 above)
Comment (Wolmin) [reply testnet/lighthouse/validator.toml]: Done

File: mainnet/lighthouse/lighthouse.toml (hunk 8 above)
Comment (Wolmin) [reply mainnet/lighthouse/lighthouse.toml:14]: Done

File: testnet/lighthouse/lighthouse.toml (hunk 7 above)
Comment (Wolmin) [reply testnet/lighthouse/lighthouse.toml:14]: Done

**Comment by mxmar** [comment]
> Thanks @Wolmin
> 
> After our discussion on Discord, I'm approving this PR. We need to solve IP flags soon: https://app.clickup.com/t/861mz2vfj

https://github.com/lukso-network/network-configs/pull/112

**Pull request merged**